components/builder-api/habitat/default.toml

```diff
@@ -27,3 +27,10 @@ jobsrv_enabled          = true
 [segment]
 url       = "https://api.segment.io"
 write_key = ""
+
+[s3]
+backend = "Minio"
+key_id = "AWSKEYIDIBELIEVE"
+secret_key = "wellIwanttobelieveanywaysbutwhoknowssincethisissecret"
+endpoint = "http://localhost:9000"
+bucket_name = "habitat-builder-dev-artifact-store"
```

Comment:
Minor - would be nice to change to "habitat-builder-local-artifact-store" (then we only need to change the key/secret for on-prem installations)

Reply:
Actually, "habitat-builder-artifact-store.default" might be a better choice.  Then, we can just replace "default" with the environment name for acceptance / live.  Dev and on-prem both just use the "default" nomenclature.  This would match up with how we are identifying services.

Reply:
We can definitely change this to something like that, I haven't tested buckets with `.` characters in them, will give that a shot. Though as a note in context to a comment further down the page. If this bucket exists in S3, that naming scheme won't really work.